test/components/DateInput_spec.jsx

```diff
@@ -69,6 +69,47 @@ describe('DateInput', () => {
       );
     });
 
+    describe('screen reader message', () => {
+      const id = 'date';
+      const screenReaderMessage = 'My screen reader message';
+      const screenReaderMessageName = `DateInput__screen-reader-message-${id}`;
+      const screenReaderMessageId = `#${screenReaderMessageName}`;
```

Comment:
Technically the ID is line 75, and this is the "selector" - could we improve the var names a bit?